.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Docker image
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Would it be beneficial to run on pull requests and tags and then limit the push to just tags?  I think if you did that might be clearer and simpler to put the pull request build into separate file.

Reply:
May as well I guess.